test/unit/api/ws/workers/peersUpdateRules.js

```diff
@@ -205,16 +238,49 @@ describe('PeersUpdateRules', function () {
 
 			it('should be ok to remove peer using different connection id', function (done) {
 				peersUpdateRules.remove(validPeer, 'different connection id', function (err) {
-					expect(err).to.be.null;
+					expect(err).to.be.undefined;
 					done();
 				});
 			});
 
-			it('should return an error when invoked with valid arguments but received error from server', function (done) {
+			it('should return the error from server when invoked with valid arguments and received error code', function (done) {
```

Comment:
`should return an error from server when invoked with valid arguments and received error code`